rust-runtime/aws-smithy-runtime/Cargo.toml

```
@@ -25,12 +25,14 @@ bytes = "1"
 http = "0.2.8"
 http-body = "0.4.5"
 pin-project-lite = "0.2.7"
+fastrand = "1.9.0"
```

Looks like most other things in the project are on 1.4
```suggestion
fastrand = "1.4.0"
```
https://github.com/awslabs/smithy-rs/blob/main/design/src/rfcs/rfc0021_dependency_versions.md